### PR TITLE
Add current sponsors

### PR DESCRIPTION
**Pull request by SepidehAlassi**

### CODE REVIEW

File: content/pages/2023/current_sponsors.md [hunk 1]
```diff
@@ -9,7 +9,27 @@ slug: sponsors_2023
...
+            technicians from almost 60 nations, sharing the passion for
+            developing and bringing new and effective medicines to patients.
+          </td>
+    </tr>
+</table>
+
 
 ## Become a Sponsor
-See guidelines on how to become a sponsor [here](sponsoring.md).
+See guidelines on how to become a sponsor [here](sponsoring.html).
```
Comment (Kislovskiy) [inline content/pages/2023/current_sponsors.md:35]: @SepidehAlassi, I think `sponsoring.html` doesn't exist, however the `sponsoring.md` does: https://github.com/euroscipy/euroscipy.github.io/blob/feature/sponsoring/content/pages/2023/sponsoring.md

File: content/pages/2023/sponsors.md [hunk 2]
```diff
@@ -10,4 +10,5 @@ slug: sponsoring_2023
 ### Links
 
 - [Become a Sponsor](sponsoring.html)
+- [Current Sponsors](current_sponsors.html)
```
Comment (Kislovskiy) [inline content/pages/2023/sponsors.md:13]: This referring the file above, right? then maybe `current_sponsors.html`? https://github.com/euroscipy/euroscipy.github.io/blob/feature/sponsoring/content/pages/2023/current_sponsors.md

**Comment by Kislovskiy** [comment]
@SepidehAlassi the content looks good to me, could you check a few things?

**Review by Kislovskiy — COMMENTED** [review]
The content looks good to me, could you check a few things?

**Comment by glemaitre** [comment]
I generated the website locally and the links are properly redirecting.
Merging.

Thanks @SepidehAlassi

**Pull request merged**